README.md

```diff
@@ -27,6 +27,17 @@ In this repository, you'll find several subprojects, each of which has its own d
 
 Please visit the `scheduler` subproject first to get started.
 
+## Quickstart
+
+The quickest way to Mesos and Cook running locally is with [docker](https://www.docker.com/) and [minmesos](https://minimesos.org/). 
+
+1. Install `docker`
+2. Install `minimesos`
+3. Clone down this repo
+4. Run `make` from the project root to build all images
+5. Run `minimesos up` from the project root to start a Mesos and Cook
```

Comment:
When running this, I get:
```
Failed to run command 'up'. Condition with alias 'Mesos master not start responding' didn't complete within 60 seconds because condition com.containersol.minimesos.mesos.MesosMasterContainer$MesosClusterStateResponse was not fulfilled.
```

What version of minimesos are you using?